client/src/components/charts/DashMap.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
+import * as chart from '../../utilities/chart';
+
+require('../../../node_modules/leaflet/dist/leaflet.css');
+require('../../styles/DashMap.scss');
+
+
+export default function DashMap({ visualisation, datasets }) {
+  const chartData = chart.getChartData(visualisation, datasets);
+
+  return (
+    <div className="DashMap dashChart">
+      <Map
+        center={[0, 0]}
+        zoom={2}
+        scrollWheelZoom={false}
```

Comment:
I think Gabe makes a valid point; scrolling for map-only pages would work well while for embed maps, it may pose some usability issues: the user would have to clearly understand that scrolling on different areas does different things, while it may be clear in isolation that scrolling on a map zooms it and scrolling on a page takes the user up/down that page, together, it adds a cognitive task of first identifying where the mouse is and then determining if scrolling there would result to the desired scroll...